dev-workflow-ui/src/ch/ivyteam/workflowui/tasks/TasksDetailsIvyDevWfBean.java

```diff
@@ -158,6 +159,23 @@ private boolean canResume() {
     return isActivator || isWorker;
   }
 
+  public boolean showInfoBanner() {
```

Comment:
Have a look at TAskNotAvailableExceptionStrategy.throwException
I would only show the banner if we have the same state in which we throw a conflict exception
1. The current session is activator: task.activator().isMember(session);
2. Task is in state TaskState.DONE, TaskState.READY_FOR_JOIN, TaskState.JOINING, TaskState.JOIN_FAILED, TaskState.CREATED, TaskSTate.RESUMED, TaskState.PARKED, TaskState.DESTROYED